packages/nextjs-mf/src/plugins/AddModulesToRuntime.js

```diff
@@ -67,7 +69,11 @@ class AddModulesToRuntimeChunkPlugin {
                 }
               }
 
-              for (const module of modulesToMove) {
+              if (partialContainerModules) {
+                containers.push(...partialContainerModules);
+              }
+
+              for (const module of [...modulesToMove, ...containers]) {
```

Comment:
NIT: for performance
```suggestion
              for (const module of [].concat(modulesToMove, containers)) {
```